File: old/aerox.Runtime.Widgets/Events/CursorMoveEvent.cs
using aerox.Runtime.Math;

namespace aerox.Runtime.Widgets.Events;

public class CursorMoveEvent : Event
{
    public Vector2<float> Position;

    public CursorMoveEvent(WidgetSurface surface, Vector2<float> position) : base(surface)
    {
        Position = position;
    }
}

File: lib/rin/gui/events/CursorMoveEvent.cpp
#include "rin/gui/events/CursorMoveEvent.h"
namespace rin::gui
{

    CursorMoveEvent::CursorMoveEvent(Surface* inSurface, const Vec2<>& inPosition) : Event(inSurface), position(inPosition)
    {
    }
}
